From: popcornmix <popcornmix@gmail.com>
Date: Tue, 28 Apr 2020 17:35:07 +0100
Subject: vc4_hvs: Mark core clock as optional

This isn't required on Pi3, so don't treat as an error

Signed-off-by: popcornmix <popcornmix@gmail.com>
---
 drivers/gpu/drm/vc4/vc4_hvs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/vc4/vc4_hvs.c b/drivers/gpu/drm/vc4/vc4_hvs.c
index 59198019aa1d..d76521be2c0d 100644
--- a/drivers/gpu/drm/vc4/vc4_hvs.c
+++ b/drivers/gpu/drm/vc4/vc4_hvs.c
@@ -240,7 +240,7 @@ static int vc4_hvs_bind(struct device *dev, struct device *master, void *data)
 	hvs->regset.regs = hvs_regs;
 	hvs->regset.nregs = ARRAY_SIZE(hvs_regs);
 
-	hvs->core_clk = devm_clk_get(&pdev->dev, NULL);
+	hvs->core_clk = devm_clk_get_optional(&pdev->dev, NULL);
 	if (IS_ERR(hvs->core_clk)) {
 		dev_err(&pdev->dev, "Couldn't get core clock\n");
 		return PTR_ERR(hvs->regs);
